Header header
string ethercat_data_raw
